backend/src/routes/v0/settings.py

```
@@ -47,21 +47,21 @@ class SingleSettingResponse(BaseModel):
 ):
     """List all settings."""
     repo = SettingsRepo(db, user.id)
-    return {"settings": repo.get_all()}
+    return {"settings": await repo.get_all()}
 
 @router.get("/settings/{setting_id}", response_model=SingleSettingResponse)
 async def get_setting(
     setting_id: str,
     user: User = Depends(verify_credentials),
-    db: Session = Depends(get_db)
+    db: Session = Depends(get_async_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)